.env

```diff
@@ -1,17 +1,16 @@
 REACT_APP_VX_ENABLE_WRITE_IN_ADJUDICATION=TRUE
-REACT_APP_VX_ENABLE_ALL_ZERO_SMARTCARD_PIN_GENERATION=TRUE
+REACT_APP_VX_ENABLE_ALL_ZERO_SMARTCARD_PIN_GENERATION=FALSE
 REACT_APP_VX_DISABLE_CARD_READER_CHECK=FALSE
 REACT_APP_VX_ENABLE_LIVECHECK=TRUE
-REACT_APP_VX_DISALLOW_CASTING_OVERVOTES=FALSE
+REACT_APP_VX_DISALLOW_CASTING_OVERVOTES=TRUE
 REACT_APP_VX_ENABLE_REACT_QUERY_DEVTOOLS=FALSE
-REACT_APP_VX_ENABLE_DEV_DOCK=TRUE
+REACT_APP_VX_ENABLE_DEV_DOCK=FALSE
 REACT_APP_VX_SKIP_PIN_ENTRY=FALSE
 REACT_APP_VX_USE_MOCK_CARDS=FALSE
 REACT_APP_VX_SKIP_CVR_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_SCAN_ELECTION_HASH_CHECK=FALSE
 REACT_APP_VX_SKIP_BALLOT_PACKAGE_AUTHENTICATION=FALSE
 REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=FALSE
-REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS=FALSE
-REACT_APP_VX_CONVERTER=ms-sems
+REACT_APP_VX_DISABLE_CVR_ORIGINAL_SNAPSHOTS=TRUE
 REACT_APP_VX_SCREEN_ORIENTATION=portrait
-REACT_APP_VX_PRECINCT_REPORT_DESTINATION=smartcard
+REACT_APP_VX_PRECINCT_REPORT_DESTINATION=laser-sheet-printer
```

Comment:
laser? not thermal?

Reply:
oh @adghayes what is the right string? you said laser-sheet-printer in the doc

Reply:
thermal per https://github.com/votingworks/vxsuite/pull/3718

Reply:
updated to thermal

Reply:
oops, thanks for catching my mistake